buildpipeline/dotnet-standard-win.json

```diff
@@ -35,7 +35,7 @@
       },
       "inputs": {
         "filename": "build.cmd",
-        "arguments": "-OfficialBuildId=$(OfficialBuildId) -- /p:SignType=$(SignType) /p:MicroBuild_SigningEnabled=true",
+        "arguments": "-OfficialBuildId=$(OfficialBuildId) -- /p:SignType=$(SignType) /p:MicroBuild_SigningEnabled=true /p:OverridePackageSource=$(PB_AzureFeedUrl)",
```

Comment:
@dagood all our repo's seem to be using OverridePackageSource should we continue to use that or move the definitions to the new property?

Reply:
From the perspective of the orchestrated build, the danger here would be that the orchestrated build passes in a source to use, then this code overrides it to the default value of `PB_AzureFeedUrl`. Won't necessarily happen, but it seems like the issue we generally need to watch out for.

There are two new properties:

 * `DotNetRestoreSourcePropsPath` (source-build) Path to a file containing MSBuild code--probably not usable here.
 * `DotNetOrchestratedRestoreSource` (orchestrator) A source to use for orchestrated builds.

I think it depends on what CoreFX is restoring from `PB_AzureFeedUrl`. If it's needed during the orchestrated build in addition to what's on the orchestrated build feed, it makes more sense to me to continue with `OverridePackageSource` (and/or find a more specific name). If not, both flows can use `DotNetOrchestratedRestoreSource`,

Reply:
Actually thinking about this more for our default case of a checked in feed we shouldn't be passing it here at all we should be adding it to the feed list in dir.props.

Reply:
I think I'm missing context.  I'm not really certain what is being discussed here or what the agreement is.  I don't know what "checked in feed" refers to.  Is there something I need to change here?

Reply:
My take is that since this Azure feed is public (correct?) then it should be checked in as one of the default sources.

We do need some way to add an additional feed (potentially secured at some point down the line), but I believe that's what @weshaggard is talking about in https://github.com/dotnet/standard/pull/553#discussion_r147272383:

> This is of course a different scenario from actually being able to override it which we want also do for the orchestrated build so depending on your goal you might want or the other.